native~/Editor/src/CesiumIonSessionImpl.cpp

```diff
@@ -758,11 +785,18 @@ CesiumAsync::Future<bool> CesiumIonSessionImpl::ensureAppDataLoaded(
              this->_pAssetAccessor,
              server.apiUrl().ToStlString())
       .thenInMainThread(
-          [this, session](
+          [this, session, asyncSystem = this->_asyncSystem](
               CesiumIonClient::Response<CesiumIonClient::ApplicationData>&&
                   appData) {
             CesiumAsync::Promise<bool> promise =
-                this->_asyncSystem.createPromise<bool>();
+                _asyncSystem.createPromise<bool>();
```

Comment:
Hm, this lambda captures `asyncSystem` but uses `_asyncSystem` in the promise. Seems unintentional?

Reply:
Yes! Good catch!